lhotse/dataset/signal_transforms.py

```diff
@@ -124,12 +125,12 @@ class SpecAugment(torch.nn.Module):
     def __init__(
         self,
         time_warp_factor: Optional[int] = 80,
-        num_feature_masks: int = 1,
-        features_mask_size: int = 13,
-        num_frame_masks: int = 1,
-        frames_mask_size: int = 70,
-        max_frames_mask_fraction: float = 0.2,
-        p=0.5,
+        num_feature_masks: int = 10,
+        features_mask_size: int = 27,
+        num_frame_masks: int = 2,
+        frames_mask_size: int = 100,
+        max_frames_mask_fraction: float = 0.15,
+        p=0.9,
     ):
```

Comment:
Incidentally, if we ever end up using a large number of feature masks (e.g. 10), which might happen for long utterances, the SpecAugment code is not efficient as it would copy the data many times.  To fix this problem, it would be necessary to refactor the code, probably so that it would first accumulate a boolean mask vector in the loop, and later apply it.